redaxo/src/core/lib/error_handler.php

```diff
@@ -227,6 +238,8 @@ private static function renderWhoops($exception)
      * @param int    $errline The line of the file in which the error occured
      *
      * @throws ErrorException
+     *
+     * @return bool
```

Comment:
Die implementierung müsste noch bool returnen..

Reply:
D.h. Hier erwartet php-src dass man bool returned, machen wir aktuell aber nicht

Reply:
wurde in einem PR inzwischen gefixed

Reply:
Ich hatte nur ein `return` geändert, es gibt also noch `return;` ohne Wert.